src/help/zaphelp/contents/ui/dialogs/options/api.html

```diff
@@ -36,6 +41,17 @@
 This is not recommended except for debugging purposes as these error messages can leak information to malicious sites.<br/>
 Note that the full error details are always written to the ZAP log file. 
 
+<H3>Report permission errors via API</H3>
+
+If enabled then ZAP will report permission errors via the API, which can be used by web applications to detect the presence of ZAP.
+This is not a serious problem in a safe environment but if you are using ZAP against potentially malicious sites then you should not enable it.
+
+<H3>Do not require an API key for safe operations</H3>
+
+If enabled then the API key is not required for Views or Other operations that are considered 'safe', in other words operations that do not make any changes to ZAP. 
```

Comment:
Could be added something like "Also, preventing session data (e.g. alerts, messages, file system paths) to be accessible without an API key"?